docs/data/toolpad/core/components/dashboard-layout/DashboardLayoutBranding.js

```diff
@@ -73,6 +73,7 @@ function DashboardLayoutBranding(props) {
       branding={{
         logo: <img src="https://mui.com/static/logo.png" alt="MUI logo" />,
         title: 'MUI',
+        homeUrl: 'https://mui.com/toolpad/core/introduction',
```

Comment:
If you click this in the demo it just changes the page content to "Dashboard content for /toolpad/core/introduction".
So not sure if we should use a relative link?
Anyway this comment is just a thought - not sure what the best option would be...

Reply:
Yes, I'll switch to the relative URL since the navigation doesn't actually work in the iframe demo